source/_integrations/nextcloud.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 title: Nextcloud
-description: Instructions on how to integrate Nextcloud monitor api data into Home Assistant.
+description: Instructions on how to integrate Nextcloud monitor api data into Home Assistant. Sensors and BinarySensors will be created for most of the datapoints that the built-in Nextcloud [serverinfo app](https://github.com/nextcloud/serverinfo) provides. To see which datapoints your nextcloud instance exposes, browse to this url: `https://<your_nextcloud_url>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json`
```

Comment:
I don't think this belongs to the meta description part here.

Please adjust the documentation below instead.

Reply:
🤦‍♂️Oh, right. Should be all set now.